README.md

```diff
@@ -62,41 +66,50 @@ I write my personal thoughts on :gear: Tech & :seedling: Life in my :writing_han
 * I feel more focused with a hoodie puts on
 * I wake up at 4:40 followed with a cold shower
 * I'm a huge fan of Iron Man and Hulk
-* I'm a big fan of *Universe of Metro 2033* - mostly books and books "Daemon" and "Freedom" written by Daniel Suarez
+* I really enjoyed *Universe of Metro 2033* books series; also a big fan of two books: "Daemon" and "Freedom" written by Daniel Suarez.
 * I use ![Todoist](https://img.shields.io/badge/-Todoist-E44332?logo=Todoist&logoColor=fff) everyday
 
 ## :hourglass_flowing_sand: Things I don't know, I want to master or just improve (randomly ordered)
```

Comment:
You have mixed a lot here. Maybe you would like to extract things that you don't know from this section. Because people can assume that you don't know a lot looking on this section

Reply:
That's why the subject is: Things I don't know, I want to master or **just improve** :)